pdc_client/__init__.py

```diff
@@ -125,7 +125,10 @@ class to perform requests.
             if insecure is not None:
                 sys.stderr.write("Warning: '%s' option is deprecated; please use '%s' instead\n" % (
                     CONFIG_INSECURE_KEY_NAME, CONFIG_SSL_VERIFY_KEY_NAME))
-                ssl_verify = not insecure
+                if insecure:
+                    ssl_verify = not insecure
+                else:
+                    ssl_verify = ssl_verify or not insecure
```

Comment:
@ycheng-aa, I hope ssl_verify is the path to CA file; not only `True/Flase`.  At same time ssl_verify can not be `None`.